Repository/PhpcrOdmRepository.php

```diff
@@ -100,6 +91,27 @@ public function getTags()
     /**
      * {@inheritdoc}
      */
+    public function reorderNode($sourcePath, $position)
+    {
+        $document = $this->getDocument($sourcePath);
+        $node = $this->getManager()->getNodeForDocument($document);
+        $parent = $node->getParent();
+        $nodeNames = $parent->getNodeNames();
+        $parentDocument = $this->getManager()->find(null, $parent->getPath());
+
+        if (isset($nodeNames[$position + 1])) {
+            $this->getManager()->reorder($parentDocument, $node->getName(), $nodeNames[$position + 1], true);
+        } else {
```

Comment:
does that mean that any position beyond the end just puts the node at the end and not raise an error? i think this makes sense, but needs to be documented.